src/app/app-routing.module.ts

```diff
@@ -63,14 +63,14 @@ const routes: Routes = [
     path: '',
     component: DefaultLayoutComponent,
     children: [
-      { path: '', redirectTo: 'servers', pathMatch: 'full' },
-      { path: 'servers', component: ServersComponent },
+      { path: '', redirectTo: 'controllers', pathMatch: 'full' },
+      { path: 'controllers', component: ControllersComponent },
       { path: 'bundled', component: BundledServerFinderComponent },
-      { path: 'server/:server_id/login', component: LoginComponent },
-      { path: 'server/:server_id/loggeduser', component: LoggedUserComponent },
-      {path : 'server/:server_id/image-manager', component: ImageManagerComponent},
+      { path: 'controller/:server_id/login', component: LoginComponent },
```

Comment:
I think `server_id` should be renamed `controller_id` everywhere in the code as well.